packages/plugin-zalo-api/src/commonUtils.ts

```diff
@@ -1,119 +1,126 @@
-import * as dotenv from "dotenv";
```

Comment:
This plugin don't related the timeclock plugin. Please remove this unnecessary changes.